docs/quick.md

```diff
@@ -137,26 +160,42 @@ follwing steps:
    
    Finally the CSPC YAML looks like the following :
    ```yaml
-       apiVersion: cstor.openebs.io/v1
-       kind: CStorPoolCluster
-       metadata:
-         name: cspc-stripe
-         namespace: openebs
-       spec:
-         pools:
-           - nodeSelector:
-               kubernetes.io/hostname: "worker1"
-             dataRaidGroups:
-             - blockDevices:
-                 - blockDeviceName: "blockdevice-10ad9f484c299597ed1e126d7b857967"
-             poolConfig:
-               dataRaidGroupType: "stripe"
+   apiVersion: cstor.openebs.io/v1
+   kind: CStorPoolCluster
+   metadata:
+     name: cspc-stripe
```

Comment:
The word "stripe" in the cluster name might confuse the users to think that the data is stripped across the nodes. Can we just keep this as "cstor-storage"